src/main.go

```diff
@@ -160,6 +160,9 @@ func setSessionVariable(db *sql.DB) {
 	if _, err := db.Exec("SET @@tidb_enable_pseudo_for_outdated_stats=false"); err != nil {
 		log.Fatalf("Executing \"SET @@tidb_enable_pseudo_for_outdated_stats=false\" err[%v]", err)
 	}
+	if _, err := db.Exec("SET @@tidb_enable_analyze_snapshot=1"); err != nil {
```

Comment:
Could you please add the related comment here, similar to the one in this PR:

> Previously, we use SI for analyze request. As RC may ignore some txnLock if the secondary keys commit havn't been finished. However, it will also meet txn conflict if we directly use maxts with SI isolevel.

> Thus we decided to use tidb_enable_analyze_snapshot to control the analyze isolevel behavior, and make it enabled for mysql-test